cmd/draft/init.go

```diff
@@ -133,6 +136,10 @@ func (i *initCmd) run() error {
 			}
 		}
 
+		basedomain := ""
+		registryURL := ""
```

Comment:
I'm assuming the same change done on line 168 and 173 also need to be done on line 148?